reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -464,7 +555,7 @@ class StaticHtmlReporter : Reporter {
             td {
                 if (row.scopes.isNotEmpty()) {
                     ul {
-                        row.scopes.entries.sortedWith(compareBy({ it.value.isNotEmpty() }, { it.key })).forEach {
+                        row.scopes.entries.sortedWith(compareBy({ it.value.isNotEmpty() }, { it.key })).forEach { it ->
```

Comment:
`it` is the default name so it should not be `assigned explicitly`.

Reply:
if I remove the "it" param from .forEach { it ->, the compiler tells me to add the explicit parameter to the outer lambda. This is the result.